lib/manage-prototype-handlers.js

```diff
@@ -277,7 +277,7 @@ function getTemplatesInstallHandler (req, res) {
 
   if (templateConfig) {
     res.render(getManagementView('template-install.njk'), {
-      currentPage: 'Templates',
+      currentPage: 'Create page from template',
```

Comment:
HMRC have decent guidance on this that makes sense to me:
https://design.tax.service.gov.uk/hmrc-design-patterns/page-title/
'the same h1 as the page'
so here that would be 
`Create new [template name]`